### PR TITLE
Initial ui

### PR DESCRIPTION
**Pull request by moonejon**

### CODE REVIEW

File: views/index.handlebars [hunk 1]
```diff
@@ -1,20 +1,71 @@
...
-        </form>
+        </div>
+
+        <!-- Modal -->
+        <div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle"
+          aria-hidden="true">
+          <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="Idea Details" id="exampleModalCenterTitle">Modal title</h5>
```
Comment (mjgardner) [inline views/index.handlebars:41]: Are these the right classes for this header? Also, "Modal title" sounds pretty generic.

File: views/index.handlebars [hunk 2]
```diff
@@ -1,20 +1,71 @@
...
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                <form class="clearfix mb-4" action="POST">
+                  <div class="form-group">
+                    <label for="example-text">Example Text</label>
```
Comment (mjgardner) [inline views/index.handlebars:49]: Again, a generic label. "Example Text"?

File: views/index.handlebars [hunk 3]
```diff
@@ -1,20 +1,71 @@
...
+                    <input type="text" id="example-text" class="form-control" aria-describedby="example-text"
+                      placeholder="Enter Example Text">
+                  </div>
+                  <div class="form-group">
+                    <label for="example-description">Example Description</label>
```
Comment (mjgardner) [inline views/index.handlebars:54]: Generic label

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -5,6 +5,12 @@
   <meta charset="utf-8">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css">
   <link rel="stylesheet" href="/styles/styles.css" media="screen" title="no title" charset="utf-8">
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
```
Comment (mjgardner) [inline views/layouts/main.handlebars:8]: I think we're going to need the full jQuery for the AJAX stuff.

File: views/index.handlebars [hunk 5]
```diff
@@ -1,20 +1,71 @@
...
+            <span class="navbar-toggler-icon"></span>
+          </button>
+        
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav mr-auto">
+              <li class="nav-item active">
+                <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Trending</a>
```
Comment (mjgardner) [inline views/index.handlebars:18]: Is this eventually going to a separate page?

**Review by mjgardner — CHANGES_REQUESTED** [review]
See my inline comments

File: views/index.handlebars (hunk 1 above)
Comment (moonejon) [reply views/index.handlebars:41]: the classes are correct, i believe. Those are bootstrap classes. The generic example text stuff is stuff that I need to replace. I'll do that this evening.

File: views/layouts/main.handlebars (hunk 4 above)
Comment (moonejon) [reply views/layouts/main.handlebars:8]: Very true... forgot about that

File: views/index.handlebars (hunk 5 above)
Comment (moonejon) [reply views/index.handlebars:18]: Yes, it is.

File: views/index.handlebars (hunk 1 above)
Comment (mjgardner) [reply views/index.handlebars:41]: It's hidden by a scrollbar in GitHub's review interface, but I was asking about the `<h5 class="Idea Details" ...>` classes.

Comment (moonejon) [reply views/index.handlebars:41]: Nope... that was a mistake I made when I was tired. Thanks for catching that.

**Review by mjgardner — APPROVED** [review]

**Pull request merged**